src/authV2/azext_authV2/azext_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
my ask is to remove the below - since anything in extensions from CLI perspective is considered preview so wouldn't recommend overriding this.